source/includes/_files.md

```diff
@@ -0,0 +1,209 @@
+# Files
+
+On the OpenDataSoft platform, files can be used as data source to create new datasets, and as dataset attachments, to add extra informations to datasets.
+
+## The file object
+
+> Example file object
+
+```json
+{
+    "file_id": "cheese_data.csv",
+    "url": "odsfile://cheese_data.csv",
+    "filename": "Cheese Data.csv",
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+The file object contains:
+
+* a unique identifier
+* a URL for use in the platform
+* the file name
+* a set of file properties
+* a creation date
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`file_id` <br> *string* | Unique identifier for the file
+`url` <br> *string* | Generated internal URL, using the `odsfile` scheme
+`filename` <br> *string* | Human readable name of the file
+`properties` <br> *object* | Set of file properties, such as its mimetype
+`created` <br> *string* | Time at which the file was created
+
+## List all files
+
+This endpoint lists all files available to the user who performs the request. A file is available to the requesting user if either the user uploaded the file themselves, or if the user has sufficient permissions to edit a dataset where the file is used, or if the user is a domain administrator who has sufficient permissions to edit all datasets of the domain.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/
+```
+
+> Example response
+
+```json
+[
+    {...},
+    {
+        "file_id": "cheese_data.csv",
+        "url": "odsfile://cheese_data.csv",
+        "filename": "Cheese Data.csv",
+        "properties": {
+            "mimetype": "text/csv"
+        },
+        "created": "2017-06-07T15:16:05.701266+00:00"
+    },
+    {...}
+]
+```
+
+## List one file
```

Comment:
Should be something like 'Retrieve one file informations'

Reply:
:+1: